util/install.sh

```diff
@@ -102,6 +148,15 @@ case $(uname) in
         #elif [[ -e /etc/redhat-release ]]; then
         #    # CentOS/RedHat
         #    os=RedHat
+        else # unrecognized Linux distro
+            os=
+            # try to grab a Linux distro identifier
+            {
+                set -o pipefail
+                hint=$(lsb_release -i | cut -f2) ||
```

Comment:
Good point.  The intention was to then set hint empty and just display a warning without it.  It seems the second line may fail if `redhat-release` doesn't exist.  I'll bulletproof it.